src/pages/NewsList/NewsList.tsx

```diff
@@ -1,10 +1,13 @@
 import { useState, useEffect } from 'react';
 
+import placeholder from '../../assets/placeholder.svg';
+
 interface NewsListProps {
 	title: string,
 	summary: string,
 	time_published: string,
 	authors: string,
+	banner_image: string | null,
```

Comment:
You can also make the banner_image optional (`banner_image?: string`). But I understand, that now it matches the current state of the API.